controller/cache/info.go

```diff
@@ -466,6 +466,17 @@ func populatePodInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 		res.Info = append(res.Info, v1alpha1.InfoItem{Name: common.PodRequestsMEM, Value: strconv.FormatInt(MemoryReq.MilliValue(), 10)})
 	}
 
+	if pod.Status.PodIP != "" {
+		res.Info = append(res.Info, v1alpha1.InfoItem{Name: "Pod IP", Value: pod.Status.PodIP})
+	}
+	if len(pod.Status.PodIPs) > 0 {
```

Comment:
i think it's better if we show the pod IP when we hover over a certain pod and not the pod IPs. We can show the Pod IPs when hovering over parent resource if they have come from it, for eg. Deployment.